moabb/datasets/physionet_mi.py

```diff
@@ -96,15 +95,27 @@ def _load_one_run(self, subject, run, preload=True):
         if get_config('MNE_DATASETS_EEGBCI_PATH') is None:
             set_config('MNE_DATASETS_EEGBCI_PATH',
                        osp.join(osp.expanduser("~"), "mne_data"))
-        raw_fname = eegbci.load_data(subject, runs=[run], verbose='ERROR',
+        raw_fname = eegbci.load_data(subject,
+                                     runs=[run],
+                                     verbose='ERROR',
                                      base_url=BASE_URL)[0]
         raw = read_raw_edf(raw_fname, preload=preload, verbose='ERROR')
         raw.rename_channels(lambda x: x.strip('.'))
         raw.rename_channels(lambda x: x.upper())
-        raw.rename_channels({'AFZ': 'AFz', 'PZ': 'Pz', 'FPZ': 'Fpz',
-                             'FCZ': 'FCz', 'FP1': 'Fp1', 'CZ': 'Cz',
-                             'OZ': 'Oz', 'POZ': 'POz', 'IZ': 'Iz',
-                             'CPZ': 'CPz', 'FP2': 'Fp2', 'FZ': 'Fz'})
+        raw.rename_channels({
+            'AFZ': 'AFz',
+            'PZ': 'Pz',
+            'FPZ': 'Fpz',
+            'FCZ': 'FCz',
+            'FP1': 'Fp1',
+            'CZ': 'Cz',
+            'OZ': 'Oz',
+            'POZ': 'POz',
+            'IZ': 'Iz',
+            'CPZ': 'CPz',
+            'FP2': 'Fp2',
+            'FZ': 'Fz'
+        })
```

Comment:
Same here, it is better to keep the code compact for such function call. You could revert this change.